x-pack/plugins/siem/server/lib/hosts/elasticsearch_adapter.ts

```diff
@@ -104,20 +104,32 @@ export const formatHostEdgesData = (
     }
   );
 
-const formatHostItem = (fields: ReadonlyArray<string>, bucket: HostAggEsItem): HostItem =>
-  fields.reduce<HostItem>((flattenedFields, fieldName) => {
+const formatHostItem = (fields: ReadonlyArray<string>, bucket: HostAggEsItem): HostItem => {
+  const a = fields.reduce<HostItem>((flattenedFields, fieldName) => {
```

Comment:
Did you want this `const a` here? Was this juts for testing? You know we love our implicit returns ;-)

Reply:
Another good 🎣 -- cleaned up and made implicit return!